### DISCUSSION TITLE
Timeline for Summit 2022

### DISCUSSION THREAD

**Discussion by claredillon** [InnerSourceCommons/InnerSourceMarketing#178] [General]
Hi All,

An action from a previous marketing group was to share the planned timelines for InnerSource Summit 2022. Here we go:

**Timeline for InnerSource Summit 2022**
Target summit dates: Wed Nov 16th - Thur Nov 17th

22 AUG - CFP Opens
19 SEP - CFP Closes [Noting this is the Monday after the LF OSS Summit in Dublin]
28 SEP - Any keynotes required confirmed
03 OCT - Early Bird Registration opens
03 OCT - Speaker confirmations published
12 OCT - Full Schedule published
16 NOV / 17 NOV - Summit Live

From previous experience, the one thing we wanted to change from last year is to leave more time between close of CFP and announcing full schedule - so we have pulled forward the CFP open date as a result. 

This is to allow for the full speaker confirmation process:
- assessment of the CFP submissions
- final proposal for the schedule
- confirmation of speaker participation
- creation of the promotional material

**Comment by voborgus** [comment]
Great plan! I highly welcome the idea to have space between the CFP closing and the final schedule. As for me, the dates suit well, thanks @claredillon

**Comment by spier** [comment]
@claredillon is this a discussion that we could close now?
